src/main/java/org/janelia/saalfeldlab/n5/N5Writer.java

```diff
@@ -162,4 +162,20 @@ public default void createDataset(
 			final String pathName,
 			final DatasetAttributes datasetAttributes,
 			final DataBlock<T> dataBlock) throws IOException;
+
+	/**
+	 * Deletes the block at {@code gridPosition}
+	 *
+	 * @param pathName dataset path
+	 * @param gridPosition position of block to be deleted
+	 * TODO is this exception necessary? Not for N5FS
+	 * @throws IOException
+	 *
+	 * TODO what is a good return value?
+	 * @return true if deletion was successful
```

Comment:
Perhaps the TODOs can be removed here

Reply:
The first one, yes. For the second one, we should probably still decide on the return value. Should anything be returned at all? Maybe just `void` is good?

Reply:
How about returning `false` if the block didn't exist, otherwise attempting to delete it and returning `true` if successful or throwing an `IOException` if not?